concent_api/api-integration-force-get-task-result-test.py

```diff
@@ -125,8 +175,9 @@ def main():
         PROVIDER_PRIVATE_KEY,
         CONCENT_PUBLIC_KEY,
         headers = {
-            'Content-Type': 'application/octet-stream',
-            'concent-client-public-key': b64encode(PROVIDER_PUBLIC_KEY).decode('ascii')
+            'Content-Type':                     'application/octet-stream',
+            'concent-client-public-key':        b64encode(PROVIDER_PUBLIC_KEY).decode('ascii'),
+            'concent-other-party-public-key':   b64encode(REQUESTOR_PUBLIC_KEY).decode('ascii')
```

Comment:
You don't have to pass 'concent-other-party-public-key' to `receive`, only to `send`, and only when communication starts (first message in each use case).